rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/kustomize/KustomizeBakeManifestRequest.java

```diff
@@ -26,4 +26,5 @@
 public class KustomizeBakeManifestRequest extends BakeManifestRequest {
   private Artifact inputArtifact;
   private String kustomizeFilePath;
+  private boolean isEnableHelm;
```

Comment:
Isn't the convention to name this enableHelm, and then the accessor is `isEnableHelm()`?
```suggestion
  private boolean enableHelm;
```

Reply:
you are right.